From 13b525667a6ab7af2354ffd6208e490349669798 Mon Sep 17 00:00:00 2001
From: Andrei Gherzan <andrei@gherzan.ro>
Date: Tue, 20 Aug 2013 16:15:15 +0300
Subject: [PATCH 2/3] Libraries and headers from ffmpeg are installed in /usr.

Don't search for libraries and headers in /usr/local.

Upstream-Status: Inappropriate [embedded specific]
Signed-off-by: Andrei Gherzan <andrei@gherzan.ro>

Updated patch to apply to c0dd9502ed2c43c487674939195c69680f3d98b0 revision

diff --git a/Makefile b/Makefile
index 65f7637..5dc1832 100644
--- a/Makefile
+++ b/Makefile
@@ -1,9 +1,9 @@
 
 CFLAGS+=-std=c++0x -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -DTARGET_POSIX -DTARGET_LINUX -fPIC -DPIC -D_REENTRANT -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -DHAVE_CMAKE_CONFIG -D__VIDEOCORE4__ -U_FORTIFY_SOURCE -Wall -DHAVE_OMXLIB -DUSE_EXTERNAL_FFMPEG  -DHAVE_LIBAVCODEC_AVCODEC_H -DHAVE_LIBAVUTIL_OPT_H -DHAVE_LIBAVUTIL_MEM_H -DHAVE_LIBAVUTIL_AVUTIL_H -DHAVE_LIBAVFORMAT_AVFORMAT_H -DHAVE_LIBAVFILTER_AVFILTER_H -DHAVE_LIBSWRESAMPLE_SWRESAMPLE_H -DOMX -DOMX_SKIP64BIT -ftree-vectorize -DUSE_EXTERNAL_OMX -DTARGET_RASPBERRY_PI -DUSE_EXTERNAL_LIBBCM_HOST
 
-LDFLAGS+=-L./ -ldbus-1 -lc -lWFC -lGLESv2 -lEGL -lbcm_host -lopenmaxil -lfreetype -lz -Lffmpeg_compiled/usr/local/lib/
+LDFLAGS+=-L./ -ldbus-1 -lc -lWFC -lGLESv2 -lEGL -lbcm_host -lopenmaxil -lfreetype -lz -Lffmpeg_compiled/usr/lib/
 
-INCLUDES+=-I./ -Ilinux -Iffmpeg_compiled/usr/local/include/
+INCLUDES+=-I./ -Ilinux -Iffmpeg_compiled/usr/include/
 
 DIST ?= omxplayer-dist
 
@@ -70,5 +70,5 @@ dist: omxplayer.bin
 	cp omxplayer omxplayer.bin $(DIST)/usr/bin
 	cp COPYING $(DIST)/usr/share/doc/
 	cp README.md $(DIST)/usr/share/doc/README
-	cp -a ffmpeg_compiled/usr/local/lib/*.so* $(DIST)/usr/lib/omxplayer/
+	cp -a ffmpeg_compiled/usr/lib/*.so* $(DIST)/usr/lib/omxplayer/
 	tar -czf omxplayer-dist.tar.gz $(DIST)
diff --git a/Makefile.ffmpeg b/Makefile.ffmpeg
index 3b31a82..5f30b51 100644
--- a/Makefile.ffmpeg
+++ b/Makefile.ffmpeg
@@ -248,5 +248,5 @@ checkout:
 
 install:
 	cd ffmpeg; make -j9 DESTDIR="$(WORK)/ffmpeg_compiled" install
-	$(HOST)-strip ffmpeg_compiled/usr/local/lib/*.so
+	$(HOST)-strip ffmpeg_compiled/usr/lib/*.so
 
